packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx

```diff
@@ -143,10 +143,10 @@ export class CalendarMonth extends BaseComponent<ICalendarMonthProps, ICalendarM
               {dateTimeFormatter.formatYear(navigatedDate)}
             </div>
           ) : (
-              <div className={css('ms-DatePicker-currentYear js-showYearPicker', styles.currentYear)}>
-                {dateTimeFormatter.formatYear(navigatedDate)}
-              </div>
-            )}
+            <div className={css('ms-DatePicker-currentYear js-showYearPicker', styles.currentYear)}>
```

Comment:
Not sure why this change is here?

Reply:
we didn't do that change, not sure why it's showing up there.

Reply:
Can you try and revert it? Otherwise beachball will think you need a change file for OUFR. You might need to check out the file from master on the command line (instead of manually editing) then commit with the `-n` option to avoid running prettier.
 ```
git checkout master -- packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx
git commit -a -m "Revert file" -n
```